### ISSUE TITLE
How to get the true labels corresponding to 'net' within the graph?

### ISSUE THREAD

**Issue by zhao62** [tflearn/tflearn#840]
with tf.Graph().as_default():
    net = tflearn.input_data(shape=[None, 27, 27, 1])
    **# As I understand, a mini-batch of 'train_data' is fed into 'net'**
    **# How can I get the true labels (train_label) corresponding to 'net' here ?**
    net = tflearn.fully_connected(net, 128, activation='relu')
    net = tflearn.fully_connected(net, 2, activation='softmax')
    mom = tflearn.Momentum(0.1, lr_decay=0.1, decay_step=7200, staircase=True)
    net = tflearn.regression(net, optimizer=mom, loss='categorical_crossentropy')
    model = tflearn.DNN(net, tensorboard_verbose=0,clip_gradients=0.)
    model.fit(train_data, train_label, n_epoch=10,
              batch_size=128, snapshot_epoch=False, snapshot_step=None, shuffle=True, run_id='nn')

**Comment by aymericdamien** [comment]
You can access the label placeholder tensor this way:
```python
with tf.Graph().as_default():
net = tflearn.input_data(shape=[None, 27, 27, 1])
net = tflearn.fully_connected(net, 128, activation='relu')
net = tflearn.fully_connected(net, 2, activation='softmax')
mom = tflearn.Momentum(0.1, lr_decay=0.1, decay_step=7200, staircase=True)
reg = tflearn.regression(net, optimizer=mom, loss='categorical_crossentropy')
labels = reg.placeholder
```

**Comment by zhao62** [comment]
Thank you very much. I have another question.

I want to try some kinds of loss function using net, labels and some other parameters. 

Can I get the labels before tflearn.regression? Or how can I do that?
